src/biogeochem/CNPrecisionControlMod.F90

```diff
@@ -24,7 +24,7 @@ module CNPrecisionControlMod
   real(r8), public :: ccrit    =  1.e-8_r8              ! critical carbon state value for truncation (gC/m2)
   real(r8), public :: cnegcrit = -6.e+1_r8              ! critical negative carbon state value for abort (gC/m2)
   real(r8), public :: ncrit    =  1.e-8_r8              ! critical nitrogen state value for truncation (gN/m2)
-  real(r8), public :: nnegcrit = -6.e+0_r8              ! critical negative nitrogen state value for abort (gN/m2)
```

Comment:
Is this an intentional change?

Reply:
I did not touch this file, so it came in with the merge.

Reply:
OK, I thought that it could've been my fault. And it is.

So I'm going to check with @olyson and we'll decide if we should keep this or leave this change out.

Reply:
OK, heard from @olyson and we should back this change out. Since, it's my fault I can do that. It's won't affect testing, except it will change namelists, so technically it would be good to resend the tests so that the namelists get done again.

Reply:
OK, reversing what I just said, this was important for the TRENDY simulations, so we should leave this in.